### ISSUE TITLE
GUI for rounding values

### ISSUE THREAD

**Issue by pbtsrc** [korapp/plasma-homeassistant#37]
Since the numbers after the dot in humidity or pressure do not make much sense, it would be very useful to be able to round values.

**Comment by Bram-diederik** [comment]
Yes please. I have an download/upload sensor of the fritzbox with many decimals.

**Comment by MitchinWilts** [comment]
Yes please, for me too. Rounding to a number of decimal places would be very useful. I use an Apollo AIR-1 for measuring air quality. Changes between Integer and Floating-point and back again is creating rounding errors. e.g. 4.4 gets rounded to 4.4000000009536 (number of zeros may be incorrect).

**Comment by tbertels** [comment]
A workaround is to create a sensor [template](https://www.home-assistant.io/integrations/template) helper in Home Assistant (HA) with
`{{ (states('sensor.sensor_id')) | round(1) }}` as its value
where sensor_id is the id of the sensor for which you want to round the value and 1 the number of digits to round to.

But this is clearly a bug in HA (or ESPHome) itself, as you can see the same conversion error when you check the value of the sensor in the HA developer tools.

**Comment by korapp** [comment]
A poll #82 has been created to gather feedback on the preferred approach to numeric value formatting.

**Comment by tbertels** [comment]
See also https://github.com/home-assistant/core/issues/148684.
